google/cloud/bigquery/v2/minimal/internal/job_request.cc

```diff
@@ -92,8 +92,8 @@ std::string ListJobsRequest::DebugString(absl::string_view name,
       .StringField("project_id", project_id_)
       .Field("all_users", all_users_)
       .Field("max_results", max_results_)
-      .Field("min_creation_time", min_creation_time_)
-      .Field("max_creation_time", max_creation_time_)
+      .Field("min_creation_time", min_creation_time_.value())
```

Comment:
Calling `.value()` if the optional is not set will result in a crash.  @devbww is working on making this easier (see #11228).